src/main/java/aztech/modern_industrialization/items/armor/GraviChestPlateItem.java

```diff
@@ -48,14 +54,16 @@ public GraviChestPlateItem(Properties settings) {
 
     @Override
     public ItemAttributeModifiers getDefaultAttributeModifiers(ItemStack stack) {
-        ItemAttributeModifiers modifiers = this.getStoredEnergy(stack) > 0 ? defaultModifiers.get() : ItemAttributeModifiers.EMPTY;
         if (this.getStoredEnergy(stack) > 0 && this.isActivated(stack)) {
-            modifiers = modifiers.withModifierAdded(
-                    NeoForgeMod.CREATIVE_FLIGHT,
-                    new AttributeModifier(MI.id("gravichestplate_flight"), 1, AttributeModifier.Operation.ADD_VALUE),
-                    EquipmentSlotGroup.CHEST);
+            return ItemAttributeModifiers.builder()
+                    .add(
+                            NeoForgeMod.CREATIVE_FLIGHT,
+                            new AttributeModifier(MI.id("gravichestplate_flight"), 1, AttributeModifier.Operation.ADD_VALUE),
+                            EquipmentSlotGroup.CHEST)
+                    .build()
+                    .withTooltip(false);
         }
-        return modifiers;
+        return ItemAttributeModifiers.EMPTY;
```

Comment:
I should point out that with this, modpacks or commands that add attributes to the gravichestplate will either A) not get shown or B) cause the "When on Body:" tooltip show twice. Not sure how much of a concern this may be, though.